ports/cwalk/vcpkg.json

```
@@ -0,0 +1,6 @@
+{
+  "name": "cwalk",
+  "version-string": "1.2.4",
```

```suggestion
  "version-string": "1.2.5",
```
It might be 1.2.5,which is the latest.